### DISCUSSION TITLE
New paper on battery integration and Energy Management!

### DISCUSSION THREAD

**Discussion by DarioSlaifsteinSk** [infiniteopt/InfiniteOpt.jl#332] [Show and tell]
Hi!
I´ve been working with ```JuMP``` and ```InfiniteOpt``` for a year. I´m using it for my PhD and got my first conference paper at IECON 2023.
The topic is Energy Management Systems for multi-carrier energy systems.
You can check out the paper here:
D. Slaifstein, J. Alpízar-Castillo, A. M. Agudin, L. Ramírez-Elizondo, G. R. C. Mouli and P. Bauer, "Aging-Aware Battery Operation for Multicarrier Energy Systems," IECON 2023- 49th Annual Conference of the IEEE Industrial Electronics Society, Singapore, Singapore, 2023, pp. 1-8, doi: 10.1109/IECON51785.2023.10312455.

Thanks a lot to the forum and to @pulsipher  for the great work and the help!

**Comment by pulsipher** [comment]
Thanks for sharing and great work!
